clusters/app.ci/assets/dptp-controller-manager.yaml

```diff
@@ -143,7 +143,7 @@ spec:
         - name: GOMAXPROCS
           value: "1024"
         - name: KUBECONFIG
-          value: /var/kubeconfigs/api-ci-kubeconfig:/var/kubeconfigs/build01-kubeconfig:/var/kubeconfigs/build02-kubeconfig:/var/kubeconfigs/vsphere-kubeconfig
+          value: /var/kubeconfigs/api-ci-kubeconfig:/var/kubeconfigs/build01-kubeconfig:/var/kubeconfigs/build02-kubeconfig:/var/kubeconfigs/vsphere-kubeconfig:/var/kubeconfigs/arm01-kubeconfig
```

Comment:
Its not that easy. The arm cluster won't be able to use the x86 images we have elsewhere.
/hold